Makefile

```
@@ -0,0 +1,41 @@
+KUEUE_REPO=https://github.com/kubernetes-sigs/kueue.git
+KUEUE_TMP_PATH=/tmp/xpk_tmp/kueue
+
+KUBECTL_AMD64="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+KUBECTL_ARM="https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubectl"
+
+OS := $(shell dpkg --print-architecture)
+
+.PHONY: install install_kjob install_kueuectl install_gcloud check_python
+
+install: install_kjob
+	pip install .
+
+install_kjob: install_kueuectl
+	git clone $(KUEUE_REPO) $(KUEUE_TMP_PATH)
+	make -C $(KUEUE_TMP_PATH)/cmd/experimental/kjobctl kubectl-kjob
+	mv $(KUEUE_TMP_PATH)/cmd/experimental/kjobctl/bin/kubectl-kjob /usr/local/bin/kubectl-kjob
+	rm -rf $(KUEUE_TMP_PATH)
+
+install_kubectl: check_gcloud
```

```suggestion
install_kubectl:
```

To install kubectl we don't need gcloud.